prov/efa/test/efa_unit_test_mocks.c

```diff
@@ -155,3 +158,49 @@ int __wrap_efadv_query_device(struct ibv_context *ibv_ctx, struct efadv_device_a
 {
 	return g_efa_unit_test_mocks.efadv_query_device(ibv_ctx, attr, inlen);
 }
+
+#if HAVE_EFADV_CQ_EX
+uint32_t efa_mock_ibv_read_src_qp_return_mock(struct ibv_cq_ex *current)
+{
+	return mock();
+}
+
+uint32_t efa_mock_ibv_read_byte_len_return_mock(struct ibv_cq_ex *current)
+{
+	return mock();
+};
+
+uint32_t efa_mock_ibv_read_slid_return_mock(struct ibv_cq_ex *current)
+{
+	return mock();
+}
+
+int efa_mock_efadv_wc_read_unknown_ah(struct efadv_cq *efadv_cq, union ibv_gid *sgid)
```

Comment:
The function name can be clear like `return_error_and_set_gid`.

Reply:
Will address.